docs/data-sources/dms_rabbitmq_flavors.md

```diff
@@ -0,0 +1,143 @@
+---
+subcategory: "Distributed Message Service (DMS)"
+---
+
+# huaweicloud_dms_rabbitmq_flavors
+
+Use this data source to get the list of rabbitMQ available flavor details within HuaweiCloud.
+
+## Example Usage
+
+### Query the list of rabbitmq flavors for cluster type
+
+```hcl
+data "huaweicloud_dms_rabbitmq_flavors" "test" {
+  type = "cluster"
+}
+```
+
+### Query the rabbitmq flavor details of the specified ID
+
+```hcl
+data "huaweicloud_dms_rabbitmq_flavors" "test" {
+  flavor_id = "c6.2u4g.cluster"
+}
+```
+
+### Query list of rabbitmq flavors that available in the availability zone list
```

Comment:
make the title fo `example usage` coincident.

Reply:
done